src/detections/__init__.py

```diff
@@ -0,0 +1,54 @@
+from src.config.config import Config
+
+
+class Detection(object):
```

Comment:
This isn't clear.
Is it a detection? or is it a query/policy?
I think better to separate it.

Create a class named Query, that will be loaded from the `yml` files in `library`. and then you'll have list of queries.
On top of that you can create query library. from there can invoke the the queries.

I think it is important to build it right so we'll be able to scale it.

Reply:
Changed to Query. Each query will have a `run` method to return vulnerable workflow paths.